src/arguments/CoreInteger.ts

```diff
@@ -18,20 +19,22 @@ export class CoreArgument extends Argument<number> {
 			});
 		}
 
-		if (typeof context.minimum === 'number' && parsed < context.minimum) {
+		const { minimum, maximum } = context;
+
+		if (minimum && parsed < minimum) {
```

Comment:
```suggestion
		if (typeof minimum === 'number' && parsed < minimum) {
```

Reply:
Also apply those requests everywhere else, don't forget, `0` is falsy.

Reply:
Yeah, that's me being a dumbass.